doc/topics/releases/fluorine.rst

```diff
@@ -3,3 +3,21 @@
 ======================================
 Salt Release Notes - Codename Fluorine
 ======================================
+
+State Module Changes
+====================
+
+states.saltmod
+--------------
+The 'test' option now defaults to None. A value of True or False set here is 
+passed to the state being run and can be used to override a `test:True` option
```

Comment:
Tip: RST syntax needs two back-ticks if you want `test: True` to show up as code markdown.